### DISCUSSION TITLE
DaisyUI being forced into dark mode by system?

### DISCUSSION THREAD

**Discussion by james9226** [saadeghi/daisyui#1819] [Q&A] [answered]
Hi everyone!

Not sure what to do here - I've read the docs and but it seems that no matter what I do, DaisyUI will keep dark mode enabled even when I explicitly disable it in my tailwind.config.js 

Investigation:
- I get dark mode when I enable dark mode on my Macbook (M1). Disabling dark mode disables dark mode on my website.
- The problem is fixed by removing DaisyUI entirely, so the issue lies with DaisyUI and not elsewhere!
- I am using DaisyUI with a NextJS 13 app, if that is relevant. 

At a bit of a loss on what to do here! Appreciate any help!

tailwind.config.js:
```
const path = require('path');

module.exports = {
  purge: [
    path.join(__dirname, './pages/**/*.{js,ts,jsx,tsx,mdx}'),
    path.join(__dirname, './components/**/*.{js,ts,jsx,tsx,mdx}'),
    path.join(__dirname, './app/**/*.{js,ts,jsx,tsx,mdx}'),
  ],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {
      fontFamily: {
        satoshi: ['Satoshi', 'sans-serif'],
        inter: ['Inter', 'sans-serif'],
      },
      colors: {
        'primary-orange': '#FF5722',
      },
    },
    daisyui: {
      styled: false,
      themes: false,
      base: false,
      utils: false,
      logs: false,
      rtl: false,
      prefix: "",
      darkTheme: "light",
    },
  },
  variants: {
    extend: {},
  },
  plugins: [
    require("daisyui"),
    require('@tailwindcss/forms')
  ],
};
```

Thank you!!

**Comment by saadeghi** [comment]
`darkMode` config has nothing to do with daisyUI themes. It's only for Tailwind's `dark:` prefix which applies `@media (prefers-color-scheme: dark)` for a specific style.  
If you only want `light` theme on daisyUI, do this:
```js
module.exports = {
  plugins: [
    require('daisyui'),
  ],
  daisyui: {
    themes: false
  },
  //...
}
```

From the docs: https://daisyui.com/docs/config/#themes

>themes
Boolean or array (default: true)
If it's true, all themes will be included. If it's false, only light (default) theme will be available. If it's an array, only themes in the array will be included and the first theme will be the default theme.

Read more about themes: https://daisyui.com/docs/themes/

**Reply by james9226** [reply]
Hi, in my config I have already set `themes: false ` yet dark mode is still being used!

**Reply by saadeghi** [reply, marked as answer]
you put `daisyui` inside `theme` object. move it out.

**Comment by james9226** [comment]
Thank you! Sorry for being a bit of an idiot, spent 2 hours trying to figure out why it wasn't working!
